client/src/app/component/production/production-content.directive.html

```diff
@@ -21,27 +21,29 @@
         </div>
     </div>
     <div class="form-group">
-        <label for="content">Vous pouvez formater le texte en utilisant le langage Markdown. Si vous
-            ne connaissez pas vous pouvez suivre ce <a href="https://help.github.com/articles/markdown-basics/">tutoriel</a>. Le principe consiste
-            à appliquer des formats en utilisant des caractères spéciaux (* = * [] !).</label>
-                    <textarea class="form-control" rows="10" ng-model="ctrl.production.content" id="content" name="content" ng-disabled="ctrl.isLocked"
+        <label for="content">Votre texte:</small>
+        <textarea class="form-control" rows="10" ng-model="ctrl.production.content" id="content" name="content" ng-disabled="ctrl.isLocked"
                               ng-required="true"
-                              ng-maxlength="3000"></textarea>
-        <small ng-if="!frm.content.$error.maxlength">({{3000 - ctrl.production.content.length }}
+                              ng-maxlength="5000"></textarea>
+        <small ng-if="!frm.content.$error.maxlength">({{5000 - ctrl.production.content.length }}
             caractère{{ctrl.production.content.length<=2999 ? 's' :
             ''}} restant{{ctrl.production.content.length<=2999 ? 's' : ''}})
```

Comment:
You need to update 2999 => 4999
I know my code stinks too... It's better to do a directive or a filter for that but time is money :smile: